FLT/HIMExperiments/module_topology.lean

```diff
@@ -213,6 +216,105 @@ noncomputable def LinearMap.add (M : Type*) [AddCommGroup M] [Module A M] :
   map_add' _ _ := add_add_add_comm _ _ _ _
   map_smul' _ _ := (DistribSMul.smul_add _ _ _).symm
 
+/-- Basis.repr in the first variable as a linear map. -/
+noncomputable def LinearMap.basis₁ (M : Type*) (ι : Type*) [Finite ι] [AddCommGroup M] [Module A M]
```

Comment:
This is `b.coord i` so we don't need a new definition.

Reply:
Oh, thanks! I was trying to find the way to construct this map